deployment/network-operator/charts/node-feature-discovery/templates/nfd-worker-conf.yaml

```diff
@@ -0,0 +1,25 @@
+{{/*
+   Copyright 2021 NVIDIA
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.worker.configmapName }}
+  labels:
+  {{- include "node-feature-discovery.labels" . | nindent 4 }}
+data:
+  nfd-worker.conf: |
+  {{ .Values.worker.config | indent 4 }}
```

Comment:
we should actually use the configmap and  put our configs here and not under options WDYT ?

i.e override this value from the top level values.yaml in network-operator

Reply:
Good catch. Let me try it